### ISSUE TITLE
fonts: adding German ß umlaut to the chapter header and parsel fonts

### ISSUE THREAD

**Issue by entorb** [yeKcim/hpmor#2]
Could you please do me a favor and add the German umlaut "ß" (sharp "s") to the fonts used at chapter header and parsel?

Note: "ß" exists only as a small letter, in case needed as capital letter, it is replaced by "SS"

For the chapter header (capital letters) a workaround would be 
\chapter[Ein äußerst unwahrscheinlicher Tag]{Ein äusserst unwahrscheinlicher Tag}

äöüÄÖÜ are already supported:
![image](https://user-images.githubusercontent.com/59419684/166814854-37d5d4a2-bf79-4ecf-a9e0-1eee8b6b5c79.png)

Same problem for the font of "writtenNote".

**Comment by yeKcim** [comment]
Is that ok here or do you need that I commit the same thing on english repository?

**Comment by entorb** [comment]
Hi, I suggest sending this fix to the EN master repo as well, and I would cherry-pick from there.

In order to do so I suggest the following workaround:

- on github.com propose an edit for 1 file
- this creates a branch in our repo based on the fork, called patch-1 or similar
- rename the branch at `github.com/<your name>/<your repo>/branches/yours`
- now you can locally pull from your master to fetch the new fork and locally add more modifications
- I suggest keeping "unmodified" base-branches of other forks, so easily create new PRs and branches based on them
- to update the fork from he fork's master, use github.com

**Comment by entorb** [comment]
Could you take a look at this one too?
https://github.com/rrthomas/hpmor/issues/68

**Comment by entorb** [comment]
Same problem for the font of "writtenNote". Are you capable of adding the "ß" there too?

**Comment by entorb** [comment]
Hi [yeKcim](https://github.com/yeKcim), thank you so much for the font fixes! The ß for Parseltongue I just found today in this issue thread, I must have missed the notification. It looks great!!!

May I ask you for adding the "ß" to fonts lumos and Graphe_Alpha_alt too? That would be quite nice, since yesterday I finished typesetting the German version, so the biggest showstopper now is this stupid character...
